src/arkode/arkode_ls.c

```diff
@@ -2670,18 +2720,19 @@ int arkLsSolve(void* arkode_mem, N_Vector b, realtype tnow,
      So we compute rwt_mean = ||rwt||_RMS and scale the desired tolerance accordingly. */
   } else if (arkls_mem->iterative) {
 
-    N_VConst(ONE, arkls_mem->x);
-    rwt_mean = N_VWrmsNorm(ark_mem->rwt, arkls_mem->x);
+    SUNCheckCallLastErrNoRet(N_VConst(ONE, arkls_mem->x));
+    rwt_mean = SUNCheckCallLastErrNoRet(N_VWrmsNorm(ark_mem->rwt, arkls_mem->x));
     delta /= rwt_mean;
 
   }
 
   /* Set initial guess x = 0 to LS */
-  N_VConst(ZERO, arkls_mem->x);
+  SUNCheckCallLastErrNoRet(N_VConst(ZERO, arkls_mem->x));
 
   /* Set zero initial guess flag */
   retval = SUNLinSolSetZeroGuess(arkls_mem->LS, SUNTRUE);
-  if (retval != SUNLS_SUCCESS) return(-1);
+  SUNCheckCallNoRet(retval);
+  if (retval != SUN_SUCCESS) return(-1);
```

Comment:
Since this routine now returns a `SUNLsStatus`, should we change the "-1" here to an appropriate `SUNLsStatus` failure flag?